stac_pydantic/links.py

```diff
@@ -19,13 +19,29 @@ class Link(StacBaseModel):
     title: Optional[str] = None
 
     # Label extension
-    label: Optional[str] = Field(default=None, alias="label:assets")
+    label: Optional[List[str]] = Field(
+        default=None,
+        min_length=1,
+        alias="label:assets",
+        validation_alias=AliasChoices("label:assets", "label_assets", "label"),
+    )
```

Comment:
🤔 well to be honest I believe we should just remove the label attribute and let `extension` be handled with `validate_extensions` 

https://github.com/stac-utils/stac-pydantic?tab=readme-ov-file#extensions

Reply:
I don't think they are validated but we could add an option. Can you open an issue in stac-fastapi?

Reply:
Feature ready: https://github.com/stac-utils/stac-fastapi-pgstac/pull/269

Replacement for this PR: https://github.com/stac-utils/stac-pydantic/pull/184